install/environment.dev.yml

```diff
@@ -6,8 +6,5 @@ channels:
 dependencies:
   - bedtools=2.29.2
```

Comment:
Just a small comment: it's useful to sort deps, imports etc alphabetically. Maybe not so relevant here, because there are only a few entries. But it's much easier to handle if the lists get longer